docs/sources/operators-guide/secure/securing-communications-with-tls.md

```diff
@@ -113,6 +114,8 @@ Each of the components listed above support the following TLS configuration opti
 - `*.tls-enabled=<boolean>`: Enable TLS in the client.
 - `*.tls-server-name=<string>`: Override the expected name on the server certificate.
 - `*.tls-insecure-skip-verify=<boolean>`: Skip validating the server certificate.
+- `*.tls-cipher-suites=<string>`: Comma-separated list of accepted cipher suites. Refer to [Grafana Mimir configuration parameters]({{< relref "../configure/reference-configuration-parameters/index.md" >}}) for the list of supported cipher suites.
```

Comment:
```suggestion
- `*.tls-cipher-suites=<string>`: Comma-separated list of accepted cipher suites. For the list of supported cipher suites, refer to [Grafana Mimir configuration parameters]({{< relref "../configure/reference-configuration-parameters/index.md" >}}).
```

Reply:
For efficiency of reading and consistency, place the "what" before the reference.